lib/host.py

```diff
@@ -85,6 +85,31 @@ def stringify(key, value):
             return False
         return result
 
+    def create_file(self, filename, text):
+        with tempfile.NamedTemporaryFile('w') as file:
+            file.write(text)
+            file.flush()
+            self.scp(file.name, filename)
+
+    def add_xcpng_repo(self, name):
+        content = """\
+[xcp-ng-{}]
+name=XCP-ng {} Repository
+baseurl=http://mirrors.xcp-ng.org/8/{}/{}/x86_64/ http://updates.xcp-ng.org/8/{}/{}/x86_64/
```

Comment:
It would not be a lot more work to replace "8" with the first number from the release.